ui/webui/test/machine_install.py

```diff
@@ -108,7 +108,11 @@ def start(self):
 
         self.http_updates_img_port = self._serve_updates_img()
 
-        payload_cached_name, http_payload_port = self._serve_payload()
+        payload_path = os.path.join(BOTS_DIR, "./images/fedora-rawhide-anaconda-payload")
+        if not os.path.exists(payload_path):
+            raise FileNotFoundError(f"Missing payload file {payload_path}. (Maybe try running 'make payload').")
```

Comment:
Perfect - just I would be more assertive on the suggestion. 
  raise FileNotFoundError(f"Missing payload file {payload_path}; use 'make payload').")

Not strong opinion though.

Reply:
updated